.github/workflows/ci.yaml

```diff
@@ -133,13 +259,38 @@ jobs:
           path: dist/
           if-no-files-found: error
 
+  test-installation:
+    name: Test package installation in different architectures
+    needs: ["build"]
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Fetch sources
+        uses: actions/checkout@v4
+      - name: Download package
+        uses: actions/download-artifact@v3
+        with:
+          name: dist-packages
+          path: dist
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+      - name: Set up docker-buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Test Installation
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: .github/containers/test-installation/Dockerfile
+          platforms: linux/amd64,linux/arm64
+          tags: localhost/test-installation
+          push: false
```

Comment:
Just FYI, this is arm64-realted, but it only tests that the package can be installed, is more lightweight than the full testing in arm64 and it runs early in the workflow and as a parallel step, so it usually doesn't affect the total workflow run time, so I would leave it.